app/src/main/java/com/appunite/loudius/ui/reviewers/ReviewersScreen.kt

```diff
@@ -151,20 +168,25 @@ private fun ReviewersScreenContent(
 
 @Composable
 private fun ReviewersList(
-    reviewers: List<Reviewer>,
+    data: Data,
+    pullRefreshState: PullRefreshState,
     modifier: Modifier,
     onNotifyClick: (ReviewersAction) -> Unit,
+    isLoading: Boolean,
 ) {
-    LazyColumn(
-        modifier = modifier.fillMaxWidth(),
-    ) {
-        itemsIndexed(reviewers) { index, reviewer ->
-            ReviewerItem(
-                reviewer = reviewer,
-                index = index,
-                onNotifyClick = onNotifyClick,
-            )
+    Box(modifier = Modifier.pullRefresh(pullRefreshState)) {
```

Comment:
**Quesion:** Do we want to pull refresh to be possible when the list is empty like there are no reviewers? If so we need to add it to he `EmptyListPlaceholder()` too.

Reply:
It's really hard to find the proper solution. I tried to add `.verticalScroll(rememberScrollState())` in many ways, but it always up the `EmptyListPlaceholder` to the top of the screen. Do you know how to do it without such a result?
![Screenshot 2023-05-29 at 16 08 49](https://github.com/appunite/Loudius/assets/72873966/b0a1f0c7-cbbc-46b3-829d-410b83c4d565)